components/ui/ModalCreation.vue

```diff
@@ -69,6 +82,41 @@ import CheckIcon from '~/assets/images/utils/right-arrow.svg'
 import Modal from '~/components/ui/Modal.vue'
 import Chips from '~/components/ui/Chips.vue'
 
+const messages = defineMessages({
+  createProjectModalDescription: {
+    id: 'modal.create-project.description',
+    defaultMessage: 'New projects are created as drafts and can be found under your profile page.',
+  },
+  createProjectModalNameLabel: {
+    id: 'modal.create-project.label.name',
```

Comment:
Since this label is associated with the input, maybe it makes to associate ID with it too. Not that it makes much difference, but it can be helpful to translators to understand the relationships of these strings.

```suggestion
    id: 'modal.create-project.input.name.label',
```